src/pages/latest/delta-update.mdx

```diff
@@ -410,24 +410,31 @@ Here is a detailed description of the `merge` programmatic operation.
-| Target columns: `key`, `value` <br></br> Source columns:` key`, `newValue`          | `targetDeltaTable.alias("t") .merge( sourceDataFrame.alias("s"), "t.key = s.key") .whenMatched().updateAll() .whenNotMatched().insert(Map( "key" -> col("s.key"), "newValue" -> col("s.newValue"))) .execute()` | `insert` throws an error because column `newValue` does not exist in the target table.                                                                                                                                                                              | `insert` still throws an error as column `newValue` does not exist in the target table.                                                                                                                                                                             |
+| Columns                                                                             | Query (in SQL)                                                                                                                                  | Behavior without schema evolution (default)                                                              | Behavior with schema evolution                                                                                                                                                                                                                                             |
+| ----------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Target columns: `key`, `value` <br></br> Source columns:` key`, `value`, `newValue` | `sql MERGE INTO target_table t USING source_table s ON t.key = s.key WHEN MATCHED THEN UPDATE SET * WHEN NOT MATCHED THEN INSERT *`             | The table schema remains unchanged; only columns `key`, `value` are updated/inserted.                    | The table schema is changed to `(key, value, new_value)`. Existing records with matches are updated with the `value` and `new_value` in the source. New rows are inserted with the schema `(key, value, new_value)`.                                                       |
```

Comment:
I haven't found a way to properly format SQL blocks inside this array and force line breaks at the right places.